packages/bpk-component-calendar/index.js

```diff
@@ -40,4 +42,5 @@ export {
   composeCalendar,
   withCalendarState,
   themeAttributes,
+  BpkCalendarDatePropTypes,
```

Comment:
Usually we give component's prop type objects a name of `propTypes` (as you have in `BpkCalendarDate.js`). But we haven't yet come across exporting multiple prop type objects in larger components - let me get input from others in the squad before settling on that name (`BpkCalendarDatePropTypes `).